portfolio/src/main/webapp/index.html

```diff
@@ -4,18 +4,21 @@
     <meta charset="UTF-8">
     <title>Jonathan Aner's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Jonathan's Portfolio</h1>
       <h2>Welcome to my Portfolio!</h2>
+      <button onclick="addGreeting()">Fetch Greeting</button>
+      <h1><div id="quote-container"></div></h1>
```

Comment:
Why did you decide to use a `<div>` over `<span>`?

Reply:
I was following the same format I saw in the week 3 folder, but span does make sense.